contrib/vivid/template.py

```diff
@@ -0,0 +1,17 @@
+pkgname = "vivid"
+pkgver = "0.10.1"
+pkgrel = 0
+build_style = "cargo"
+hostmakedepends = ["cargo-auditable"]
+makedepends = ["rust-std"]
+pkgdesc = "Themeable LS_COLORS generator"
+maintainer = "jabuxas <jabuxas@proton.me>"
+license = "MIT AND Apache-2.0"
+url = "https://github.com/sharkdp/vivid"
+source = f"{url}/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "88db6158dad60aba66ae16f2cd1b09f515625940a33bada65da5562a03538e49"
+
+
+def post_install(self):
+    self.install_license("LICENSE-APACHE")
```

Comment:
```suggestion
```

Reply:
uhmm why? main/rust installs both licenses

Reply:
because apache doesn't have to be installed

Reply:
ok so why does rust do it

Reply:
because someone put it there when they didn't have to

Reply:
sure